docs/copilot/overview.md

```diff
@@ -21,162 +21,101 @@ With GitHub Copilot in VS Code you can:
-
-## Inline suggestions
-
-As you start writing code or code-related items (comments, test, and more), Copilot presents suggestions automatically in the editor to help you code more efficiently. For any given input, Copilot might offer multiple suggestions.
-
-GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks, and it works especially well for Python, JavaScript, TypeScript, Ruby, Go, C# and C++.
-
-Notice in the following example how Copilot suggests an implementation of the `calculateDaysBetweenDates` JavaScript function by using dimmed *ghost text*:
-
-![JavaScript ghost text suggestion.](images/overview/js-suggest.png)
-
-When you're presented with an inline suggestion, you can accept it with the `kbstyle(Tab)` key. Optionally, you can hover over the suggestion to any of the other suggestions.
-
-Rather than letting Copilot provide suggestions as you're typing, you can also use code comments to provide instructions to Copilot. By using code comments, you can be more specific the suggestions you're looking for. For example, you could specify a type of algorithm to use, or which methods and properties to add to a class.
-
-The following example shows how to instruct Copilot to create a class in TypeScript to represent a student, providing information about methods and properties:
-
-![Use code comments to let Copilot generate a Student class in TypeScript with properties and methods.](images/overview/ts-suggest-code-comment.png)
-
-## Chat features
+* To use GitHub Copilot, you must have an active subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization. Sign up for a [GitHub Copilot free trial](https://github.com/login?return_to=%2fgithub-copilot%2fsignup) in your personal account.
 
-In addition to inline suggestions, you can also get assistance from Copilot via a chat interface. You can use Copilot Chat in VS Code in different ways, depending on the context:
+Follow these steps to [Set up GitHub Copilot in VS Code](/docs/copilot/getting-started.md#set-up-vs-code-for-github-copilot).
 
-* **Chat view:** Ask Copilot for help with any task or question in the Chat view, bringing in code from your current files.
-* **Inline Chat:** Apply Copilot's AI suggestions directly to your code, staying in the flow.
-* **Quick Chat:** Bring up a Chat-powered dropdown for quick questions and suggestions.
+## Copilot tutorials
 
-To use the chat features in VS Code, install the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extension.
+* [GitHub Copilot quickstart](/docs/copilot/getting-started.md)
+* [Get started with Copilot Chat](/docs/copilot/getting-started-chat.md)
 
-> <a class="install-extension-btn" href="vscode:extension/GitHub.copilot-chat">Install the GitHub Copilot Chat extension</a>
+## Use cases for GitHub Copilot in VS Code
```

Comment:
I absolutely love this new use cases section! For even easier navigation and scanning, I wonder if a quick bulleted list that linked to the top-level categories (i.e. Code completions in the editor, Answering coding questions) would help. This way, I can quickly scan what kinds of scenarios Copilot helps with and potentially click to those I find most interesting first, without needing to scroll.

Reply:
I also think either videos or screenshots would work well. If @burkeholland or others have a video for a specific topic, that could be really cool to integrate, but given the number of scenarios we'd like to cover here, I think screenshots are fast and effective.

Reply:
Agree. It might be too distracting, I'll incorporate more videos in the individual articles.